### ISSUE TITLE
making easy

### ISSUE THREAD

**Issue by jobdiogenes** [peterwilli/Colab_SFTP#2]
Like You and others I was in the same situation about gdrive not so stable.
Or just want to access other kind of foreign data.    That could be in a  ftp, sql, etc.
One thing that stress me is to be asking for password. 
Off course, put in script is not the solution.
But, since now we can mount Gdrive, we can store password there
in a file and get then and go. 
Scripts could also check for passfile, and if not found or login fail ask the user, to mount gdrive, or to correct passwordfile.

Thanks for sharing yous ftp/colab.

**Comment by peterwilli** [comment]
Hey there,

Thanks for sharing your ideas.
Wouldn't storing a password on gdrive still force people to get/share a API key to gdrive to get the password?

**Comment by jobdiogenes** [comment]
> 

No, will not ask the password, since will read from file system.
